tests/foreman/ui_airgun/test_repository.py

```diff
@@ -320,6 +329,23 @@ def test_positive_discover_repo_via_new_product(session, module_org):
             product_name, repo_name)[0]['Name']
 
 
+@skip_if_bug_open('bugzilla', 1676642)
+@stubbed
+@tier2
+@upgrade
+def test_positive_discover_module_stream_repo_via_existing_product(session, module_org):
+    """Create repository with having module streams via repo-discovery under existing product
```

Comment:
`Create repository with module streams via repo-discovery under existing product`?

Reply:
the objective here to make sure the repository should be created from url which contains module streams. Currently, there is a bug associated with that. if you see that test name confusing, I will update it.